docs/quick-start/cli_code.md

```diff
@@ -11,7 +11,7 @@
 
 1. Compile and program your board:
 
-   Invoke `mbed compile`, and specify the name of your platform and your installed toolchain (`GCC_ARM`, `ARM`, `IAR`). For example, for the K64F platform and Arm Compiler 5 toolchain:
+   Invoke `mbed compile`, and specify the name of your platform and your installed toolchain (`GCC_ARM`, `ARM`, `IAR`). For example, for the K64F platform and ARM toolchain:
```

Comment:
Does it actually still allow ARM? Is ARM now equal to ARM6?

Because when I look at where the single source of truth may be, the mapping between Mbed CLI's supported versions and these parameter values isn't obvious. 

```
- Arm Compiler 6.13 (default ARM toolchain).
  - A paid version is available as [Arm Compiler 6.13 Professional](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler/downloads/version-6).
  - A paid version is also included in [Keil MDK 5.29](http://www2.keil.com/mdk5/529).
- [GNU Arm Embedded version 9 (9-2019-q4-major)](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads).
- [IAR Embedded Workbench 8.32.1](https://iar.com/mbed).
```